src/main/webapp/app/exercises/quiz/manage/re-evaluate/multiple-choice-question/re-evaluate-multiple-choice-question.component.ts

```diff
@@ -186,7 +157,7 @@ export class ReEvaluateMultipleChoiceQuestionComponent implements OnInit, AfterV
         const startOfThisPart = text.indexOf(answerOptionText);
         const box = text.substring(0, startOfThisPart);
         // Check if box says this answer option is correct or not
-        answer.isCorrect = box === '[x]' || box === '[X]';
+        answer.isCorrect = box === '[correct]';
```

Comment:
```suggestion
        answer.isCorrect = box === CorrectOptionCommand.identifier;
```

Reply:
what happens if someone uses capital letters? can we do a lower case based comparison here an in all other cases?

Reply:
That's a good question in general and not just in this case. I tested it now on a small example and it is not parsed correctly and the `[Correct]` tag (I tested it like that) is displayed.
I think a case insensitive parsing (for all commands) would be a nice addition at a later stage but it is not really urgent at the moment since there are buttons that add the tags correctly.
And I think that more people use the commands than typing them